biopet-extensions/src/main/scala/nl/lumc/sasc/biopet/extensions/gatk/MergeSamFiles.scala

```diff
@@ -15,13 +15,13 @@
 package nl.lumc.sasc.biopet.extensions.gatk
 
 import nl.lumc.sasc.biopet.core.BiopetCommandLineFunction
-import nl.lumc.sasc.biopet.extensions.picard.MergeSamFiles
+import nl.lumc.sasc.biopet.extensions.picard
 import org.broadinstitute.gatk.queue.function.scattergather.GatherFunction
 
 /**
  * Merges BAM files using htsjdk.samtools.MergeSamFiles.
  */
-class BamGatherFunction extends MergeSamFiles(null) with GatherFunction {
+class MergeSamFiles extends picard.MergeSamFiles(null) with GatherFunction {
```

Comment:
This change was to small to make an new PR for this. Still your right, it's cleaner to do this in a other PR/issue.